hat/assets/js/apps/Iaso/domains/orgUnits/reviewChanges/Tables/ReviewOrgUnitChangesTable.tsx

```diff
@@ -60,6 +60,11 @@ const useColumns = (
                 return <>{formatMessage(MESSAGES.newOrgUnit)}</>;
             },
         },
+        {
+            Header: formatMessage(MESSAGES.parent),
+            id: 'org_unit__parent__name',
+            accessor: 'org_unit_parent_name',
+        },
```

Comment:
i would use `LinkToOrgUnit` to display link to parent org unit, and a "--" as placeholder of no parent